src/components/LibraryContainer.tsx

```diff
@@ -352,6 +363,12 @@ export class LibraryContainer extends React.Component<LibraryContainerProps, Lib
         }
     }
 
+    setOverride(override:boolean){
+        if(this.state.shouldOverideExpandedState != override){
+        this.setState({shouldOverideExpandedState:override})
+        }
+    }
+
```

Comment:
what is this used for ?

Reply:
see the description:
`
LibraryContainer.State.shouldOverideExpandedState  also controls the rendering behavior of LibraryItems - I found that LibraryItems today use a legacy life cycle hook UNSAFE_onWillRecieveProps - to override state with props - specifically for the expanded state so that when interacting only one library item can be open at a time.
`

TLDR: it's used to set a specific rendering mode where props.expanded overrides state.expanded of library items so they close.

Reply:
ah, but your point is that this function setting the state is never called, thanks, I'll get rid of it, I ended up needing to set the state manually to avoid triggering a render, not ideal, but I think acceptable for now, I will get rid of this method.